cypress/pageobjects/virtualmachine.po.ts

```diff
@@ -27,16 +26,16 @@ interface ValueInterface {
 export class VmsPage {
   public goToList() {
     cy.visit(constants.vmPage);
-    cy.intercept('GET', '/v1/harvester/kubevirt.io.virtualmachines').as('loadVM');
-    cy.wait('@loadVM')
-    expect(cy.url().should('eq', constants.vmPage));
+
+    cy.get('nav.side-nav .list-unstyled').find('li').contains('Virtual Machines').click();
+    cy.get('main .outlet header h1').then($el => {
```

Comment:
These gets should be pointed at a class member that is assigned the css selectors. Such as
```
private sideBar = 'nav.side-nav .list-unstyled';
cy.get(this.sideBar).find
```

Reply:
Done